ui/src/rootReducer.ts

```diff
@@ -2,24 +2,40 @@ import { combineReducers, createAction } from "@reduxjs/toolkit";
 import cohortsReducer from "cohortsSlice";
 import conceptSetsReducer from "conceptSetsSlice";
 import { AnyAction, Reducer } from "redux";
+import undoable, { excludeAction } from "redux-undo";
 import * as tanagra from "tanagra-api";
 import underlaysReducer from "underlaysSlice";
+import urlSlice from "urlSlice";
+
+export const loadUserData = createAction<tanagra.UserData>("loadUserData");
+
+const undoableConfigs = {
+  initTypes: ["loadUserData"],
+  filter: excludeAction([loadUserData, "underlays/setUnderlays"]),
```

Comment:
On the other hand, it seems unnecessary to include `loadUserData` here since it's the initial state.